Tests/QuantConnect.Tests.csproj

```diff
@@ -12,7 +12,6 @@
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
```

Comment:
Guess this should remain in the csproj?

Reply:
This appeared to be the only csproj that had it and caused problems in vs2015.